supernode/daemon/mgr/gc/gc_disk.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gc
+
+import (
+	"context"
+
+	"github.com/dragonflyoss/Dragonfly/pkg/errortypes"
+	"github.com/dragonflyoss/Dragonfly/supernode/util"
+
+	"github.com/sirupsen/logrus"
+)
+
+func (gcm *Manager) gcDisk(ctx context.Context) {
+	gcTaskIDs, err := gcm.cdnMgr.GetGCTaskIDs(ctx, gcm.taskMgr)
+	if err != nil {
+		logrus.Errorf("gc disk: failed to get gc tasks: %v", err)
+		return
+	}
+
+	if len(gcTaskIDs) == 0 {
+		logrus.Warnf("gc disk: get empty gcTasks")
+		return
+	}
+
+	logrus.Debugf("gc disk: success to get gcTaskIDs(%v)", gcTaskIDs)
+	go gcm.deleteTaskDisk(ctx, gcTaskIDs)
```

Comment:
* don't run it in a different routine
* output the gc result at the end, it's useful for troubleshooting

Reply:
Done. 

And I output the gc result at the function `deleteTaskDisk `.